manifests/k/KeePassXCTeam/KeePassXC/2.7.6/KeePassXCTeam.KeePassXC.installer.yaml

```diff
@@ -19,11 +19,13 @@ FileExtensions:
 - kpdx
 - lngx
 - plgx
-ProductCode: '{AE8C6DDF-D052-4AEF-9EE3-8F354EC1530D}'
+ProductCode: '{620E353A-56BA-46ED-A729-CE4A92A14DAC}'
+ReleaseDate: 2023-08-16
+AppsAndFeaturesEntries:
+- UpgradeCode: '{88785A72-3EAE-4F29-89E3-BC6B19BA9A5B}'
 Installers:
 - Architecture: x64
-  InstallerUrl: https://github.com/keepassxreboot/keepassxc/releases/download/2.7.6/KeePassXC-2.7.6-Win64.msi
-  InstallerSha256: 14F0A77D934AE25075597FA65EE31C666A524133BF67D459891222CA85DE1C2F
+  InstallerUrl: https://github.com/keepassxreboot/keepassxc/releases/download/2.7.6/KeePassXC-2.7.6-Win64-LegacyWindows.msi

```

Comment:
Why was this changed to the legacy (Windows 7 / 8 / 8.1) installer? Other versions use the Windows 10 / 11 installer